PacklinkPro/Services/BusinessLogic/ConfigurationService.php

```diff
@@ -101,7 +101,7 @@ public function getIntegrationName()
      */
     public function getCurrentSystemId()
     {
-        return $this->storeManager->getStore()->getId();
+        return '';
```

Comment:
Just to make sure. Is this correct?

Reply:
yes, it was the original bug to use store ID